test/unit/helpers/api/integrations_helper_test.rb

```diff
@@ -230,4 +230,17 @@ class IsHttpTest < self
       refute(is_https?(nil))
     end
   end
+
+  class ProxyRulesPreview < self
+    include ProxyRulesHelper
+
+    test 'print only 1 slash when backend_api path is "/"' do
+      backend_api = FactoryBot.create(:backend_api)
+      proxy_rule = FactoryBot.create(:proxy_rule, owner: backend_api, proxy: nil)
+      backend_api_config = FactoryBot.create(:backend_api_config)
+      backend_api_config.stubs(:path).returns('/')
+
+      assert_no_match %r{//}, proxy_rules_preview(backend_api, path: backend_api_config.path)
```

Comment:
Better to assert the exact text expected, instead of using `assert_no_match`. There are two reasons for that:
1. If the implementation changes, there are more chances of this test to become always green than otherwise, meaning it will be testing nothing
2. `/some//path` is actually is valid proxy rule and would make this test to raise a false-positive.